src/components/PromQueryField.test.tsx

```diff
@@ -126,29 +141,13 @@ describe('PromQueryField', () => {
 
     // If we check the label browser right away it should be in loading state
     let labelBrowser = screen.getByRole('button');
-    expect(labelBrowser.textContent).toContain('Loading');
-  });
-
-  it('should not run query onBlur in explore', async () => {
-    const onRunQuery = jest.fn();
-    const { container } = render(<PromQueryField {...defaultProps} app={CoreApp.Explore} onRunQuery={onRunQuery} />);
-
-    const input = getByTestId(container, 'dummy-code-input');
-    expect(input).toBeInTheDocument();
-    await userEvent.type(input, 'metric');
-    input.blur();
-    expect(onRunQuery).not.toHaveBeenCalled();
-  });
```

Comment:
Do you think we should remove this test?

Reply:
reverted